docs/source/configuration.rst

```diff
@@ -212,30 +209,29 @@ be a :code:`.sql` itself.
     be a requirement that the *templating* is accurate - only so far as that
     is required to enable the *parsing* and *linting* to be helpful.
 
-Builtin Macro Blocks
-^^^^^^^^^^^^^^^^^^^^
+Dbt Project Configuration
+-------------------------
 
-One of the main use cases which inspired *sqlfluff* as a a project was `dbt`_.
-It uses jinja templating extensively and leads to some users maintaining large
-repositories of sql files which could potentially benefit from some linting.
+dbt is not the default templater for *sqlfluff* (it is Jinja). In order
+to get started using *sqlfluff* with a Dbt project you will need the following
```

Comment:
```suggestion
to get started using *sqlfluff* with a dbt project you will need the following
```

so minor I feel ashamed but while we're here

Reply:
@NiallRees - this is a project about linting. There is no shame.